gtfs/gtfs_utils/gtfs_utils/core_computations.py

```diff
@@ -187,12 +235,20 @@ def compute_trip_stats(feed: ptg.feed,
 
     # Reset index and compute final stats
     h = h.reset_index()
+
+    # Add rank to duplicate trips and add trip_id_to_date data
+    h['numeric_trip_id'] = h['trip_id'].apply(lambda s: int(s.replace('_', '')))
+    h['trip_id_rank'] = h.groupby(['route_id', 'start_time'])['numeric_trip_id'].rank(method='dense')
+    h = h.merge(trip_to_date, how='inner', on=['route_id', 'start_time', 'trip_id_rank'], validate='one_to_one')
```

Comment:
oooh, I didn't know the `validate` param, nice. 
So this will raise an error if it is not 1:1? Did this ever happen? On how many different files/dates did you check?